wsd/Admin.cpp

```diff
@@ -294,6 +298,26 @@ void AdminSocketHandler::handleMessage(const std::vector<char> &payload)
         // Let's send back the current log levels in return. So the user can be sure of the values.
         sendTextFrame("channel_list " + _admin->getChannelLogLevels());
     }
+    else if (tokens.equals(0, "updateroutetoken") && tokens.size() > 1)
+    {
+        // parse the json object of serverId to routeToken
+        Poco::JSON::Object::Ptr object;
+        if (JsonUtil::parseJSON(tokens[1], object))
+        {
+            const std::string routeToken =
+                JsonUtil::getJSONValue<std::string>(object, Util::getProcessIdentifier());
+            if (routeToken.empty())
+            {
+                LOG_ERR("Failed to update the route token, invalid serverId to routeToken json : "
+                        << tokens[1]);
+            }
+            COOLWSD::alertAllUsersInternal("updateroutetoken " + routeToken);
```

Comment:
Looks fine - I wonder if we need to store the routeToken internally for our server (?) might be nice to have it around so we can dump it with a dumpState() method (?)